src/app/services/game-play.service.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { firstValueFrom, of } from 'rxjs';
+
+import { GamePlayService } from './game-play.service';
+import { AnswerService } from './answer.service';
+import { Answer } from '../models/answer';
+
+const mockAnswer = <Answer>{ id: 1, word: 'ADEPT' };
```

Comment:
Nit-pick: I think it's better practice to get in the habit of typing the variable rather than using a cast:

```typescript
const mockAnswer: Answer = { id: 1, word: 'ADEPT' }
```